src/scaling.jl

```diff
@@ -68,7 +68,8 @@ struct MeanStdScaling <: AbstractScaling
     end
 end
 
-compute_stats(x) = (mean(x), std(x))
+# Set std to 0 using corrected=false if x is a singleton
+compute_stats(x) = (mean(x), std(x; corrected=(length(x) != 1)))
```

Comment:
maybe it's better to pass this via `MeanStdScaling` as a kwarg? since it's already exposed via `Statistics.std` anyway?

Reply:
I see where you're coming from by leaving it up to the user. I guess it would just be inconvenient in the case that came up for me. I would rather change the test case so that the data doesn't end up being one row, than add `corrected=(length(x) != 1)` to `MeanStdScaling` in the transform pipeline, in the source code.